src/lib/docker/composeFile.ts

```diff
@@ -118,6 +118,11 @@ class ComposeFile {
     let nodeCommand = node.docker.command || getDefaultCommand('c-lightning', version);
     // do not include the GRPC port arg in the command for unsupported versions
     if (grpc === 0) nodeCommand = nodeCommand.replace('--grpc-port=11001', '');
+    // do not include GRPC host arg in the command for unsupported versions
+    // only versions: v24.05 and v24.08.1 do not support the arg.
+    if (version === '24.05' || version === '24.08.1')
+      nodeCommand = nodeCommand.replace('--grpc-host=0.0.0.0', '');
+
```

Comment:
I've confirmed this change fixes the issue with older versions.

I would just suggest to move this check into `getDefaultCommand`. This is where I'd prefer to aggregate these conditions and tweaks to the commands. You could also use `isVersionBelow` instead of hard-coding multiple versions explicitly. This better reflects the reality that any version below `24.11`.